### ISSUE TITLE
Can you share this app？

### ISSUE THREAD

**Issue by zhuang0393** [PattyMosher/HomeNetworkApp#1]
I want to create vlans with this app, Can you share this app？

**Comment by PattyMosher** [comment]
Unfortunately no. I don't have IP rights to share/publish the whole application. What I did upload is just as a sample to showcase some of my programming skills. If you'd like to build something similar, it was built to work with OpenWRT and HostAPD.

**Comment by zhuang0393** [comment]
Alright，Thank you you for you prompt reply！

**Comment by PattyMosher** [comment]
Yeah, no problem. I've been wanting to create something that does something similar from scratch for myself. If/when I get around to it, I'll let you know.
